pkg/namesgenerator/names-generator.go

```diff
@@ -839,10 +839,9 @@ var (
 // formatted as "adjective_surname". For example 'focused_turing'. If retry is non-zero, a random
 // integer between 0 and 10 will be added to the end of the name, e.g `focused_turing3`
 func GetRandomName(retry int) string {
-begin:
 	name := left[rand.Intn(len(left))] + "_" + right[rand.Intn(len(right))] //nolint:gosec // G404: Use of weak random number generator (math/rand instead of crypto/rand)
 	if name == "boring_wozniak" /* Steve Wozniak is not boring */ {
-		goto begin
+		name = "boring_jobs"
```

Comment:
I should probably add here that there's no `jobs` in the list currently.

Reply:
Isn't that the point? Wozniak isn't boring, but Jobs is. In my opinion this would make clear, that wozniak is better than jobs.